packages/clay-color-picker/package.json

```diff
@@ -0,0 +1,37 @@
+{
+	"name": "clay-color-picker",
+	"version": "0.0.1",
+	"private": true,
+	"main": "lib/index.js",
+	"jsnext:main": "src/index.js",
+	"scripts": {
```

Comment:
We can add a `build` task, so when someone runs `yarn build` at the root, we can perform the build of all the components. You can see how I did it on [ClayDatePicker](https://github.com/liferay/clay/pull/1440/commits/a96bad47531059edb4262950aa19ddb2d7907de6)